index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+<div id="mainWrapper" class="wrapper">
+    <nav class="mainNav closed" id="mainNav">
+        <a href="javascript:void(0);" class="burger-icon menu-items-toggle" id="nav-itemsToggle" onclick="myFunction()">
```

Comment:
If you have an element that doesn't redirect you, but will perform an action, you can use a button for that
```suggestion
        <button class="burger-icon menu-items-toggle" id="nav-itemsToggle" onclick="myFunction()">
```